tensorbay/opendataset/HKD/loader.py

```diff
@@ -96,6 +96,14 @@ def HKD(path: str) -> Dataset:
                 image_path = os.path.join(
                     segment_path, "CropImages", image_name_template.format(csv_line.pop(0))
                 )
+                if "NaN" in csv_line:
+                    continue
+                if len(segment) != 0:
+                    if image_path == segment[-1].path:
+                        segment[-1].label.keypoints2d[0].extend(
+                            [LabeledKeypoints2D(chunked(map(float, csv_line), 2))][0]
+                        )
+                        continue
```

Comment:
Check and create the `Data` first, and then check whether to add the label. Separate the two steps.
Maybe use a variable like `last_filename` to record the previous data?